setup-scripts/install.sh

```diff
@@ -301,12 +294,7 @@ cd broker
 
 # Set up environment
 msg "Setting up your Broker" $WHITE
-npm run env-setup -- -n=$NETWORK
-
-# Install CLI
-msg "Installing the CLI" $WHITE
-npm install -g ./broker-cli
```

Comment:
Since we removed the cli (because of nvm), we fail to "install" the config on a users machine. This script now throws the error

```
[⚡ Sparkswap Installer]:  Creating random username and password
sed: can't read /root/.sparkswap/config.js: No such file or directory
```